src/ocrd/decorators/ocrd_cli_options.py

```diff
@@ -56,7 +56,7 @@ def cli(**kwargs):
         # subcommands. So we have to work around that by creating a
         # pseudo-subcommand handled in ocrd_cli_wrap_processor
         argument('subcommand', nargs=1, required=False,
-                 type=click.Choice([AgentType.PROCESSING_WORKER, AgentType.PROCESSOR_SERVER])),
+                 type=click.Choice(list(map(str, AgentType)))),
```

Comment:
Why does the previous implementation no longer work? Is it because of the updated click version?

Reply:
Click 8.2 introduced direct support for enum types (without the `list` wrapper and the `.value` function). But that is not even available on Python 3.8 or 3.9. And it does not explain why the previous setting ever worked. I don't know TBH.